src/LanguageFunctions.ts

```diff
@@ -102,19 +102,29 @@ export async function FindAllUnTranslatedText(): Promise<void> {
     if (Settings.GetConfigSettings()[Setting.UseExternalTranslationTool]) {
         findText = GetTargetStateActionNeededToken();
     } else {
-        findText = escapeStringRegexp(GetReviewToken()) + '|' + escapeStringRegexp(GetNotTranslatedToken());
+        findText = escapeStringRegexp(GetReviewToken()) + '|' + escapeStringRegexp(GetNotTranslatedToken()) + '|' + escapeStringRegexp(GetSuggestionToken());
     }
     let fileFilter = '';
     if (Settings.GetConfigSettings()[Setting.SearchOnlyXlfFiles] === true) { fileFilter = '*.xlf'; }
     await VSCodeFunctions.FindTextInFiles(findText, true, fileFilter);
 }
 
+export async function FindMultipleTargets(): Promise<void> {
+    const findText = '^\\s*<target>.*\\r*\\n*(\\s*<target>.*)+';
+    let fileFilter = '';
+    if (Settings.GetConfigSettings()[Setting.SearchOnlyXlfFiles] === true) { fileFilter = '*.xlf'; }
+    await VSCodeFunctions.FindTextInFiles(findText, true, fileFilter);
+}
+
 export function GetNotTranslatedToken(): string {
     return '[NAB: NOT TRANSLATED]';
 }
 export function GetReviewToken(): string {
     return '[NAB: REVIEW]';
 }
+export function GetSuggestionToken(): string {
```

Comment:
Future improvement: Move these to an enum like XliffTargetState or something similar?